docs/resources/container_file.md

```diff
@@ -1,5 +1,7 @@
 # lxd_container_file
 
+!> Resource `lxd_container_file` has been deprecated and will be removed. Please use [`lxd_instance_file`](../instance_file.md) instead.
```

Comment:
I've added a warning with the link to the new resource `... Please use [lxd_instance_file](../instance_file.md) instead.`. 

Can you confirm the url pointing to a new resource is correct, to ensure there is no broken links?

Reply:
I can't find any guidance on links between resources for terraform docs. Note that these get generated into what shows up on the registry at https://registry.terraform.io/providers/terraform-lxd/lxd/latest/docs/resources/container . Given that, I suspect the link is not correct since it has `.md`, but without deploying a release I know no other way to test them. I think it should be sufficient to just mention the new resource by name and let the users navigate using the sidebar.

Reply:
I agree here. 

One option would be to dive into some other Terraform provider repos and try to find an example of inter-linking within documentation. Short of that, though, just mentioning the resource name is the best way to ensure there isn't a broken link.

Reply:
I looked at a few but they either didn’t have any links or were using the `website/docs` structure which I didn’t explore.

Reply:
I had the same issue with finding any official guidance on links between resources.
I've removed the links, leaving just the warnings.